packages/demo-html/public/chat.html

```diff
@@ -45,6 +45,7 @@ <h1>this is an experimental feature</h1>
       data-tf-popover="moe6aa"
       data-tf-medium="demo-test"
       data-tf-chat
+      data-tf-notification-dot="3"
```

Comment:
At first I thought this would be a number inside the dot. But its number of days to display the dot for, right? Maybe we could find a better name? `data-tf-notification-days`?

Reply:
I am fine with any naming